### PR TITLE
Add Jump to Message link

### PR DESCRIPTION
**Pull request by Ta1al**
Added [Jump to Message] hyperlink in dump command

Update: hyperlink won't work since it's not a slash command, so I replaced it with a normal link

### CODE REVIEW

File: src/commands/mention/dump.ts [hunk 1]
```diff
@@ -14,7 +14,10 @@ const command: MentionCommand = {
       content: `💾 Guild data for server ${guildId} as of <t:${Math.floor(now / 1000)}:R>.`,
       files: [{ attachment: Buffer.from(data), name: `Countr-Dump-${guildId}-${now}.json` }],
     })
-      .then(() => reply(`📨 Guild data of ${guildId === message.guildId ? "this server" : `server with ID \`${guildId}\``} has been dumped to your DMs.`))
+      .then(msg => reply(`
+        📨 Guild data of ${guildId === message.guildId ? "this server" : `server with ID \`${guildId}\``} has been dumped to your DMs.
+        [Jump to Message](${msg.url})
```
Comment (djohts) [inline src/commands/mention/dump.ts]: message url should be modified by replacing user's id with bot's id

**Review by djohts — CHANGES_REQUESTED** [review]

File: src/commands/mention/dump.ts (hunk 1 above)
Comment (djohts) [reply src/commands/mention/dump.ts]: otherwise it will show "you dont have access to that channel/message" pop up

Comment (promise) [reply src/commands/mention/dump.ts]: there's no user id in the message link, all dms are channels and have a channel id

**Review by djohts — APPROVED** [review]

**Comment by promise** [comment]
just realized `[click me](link)` won't work since this is a normal message, not an interaction/webhook. it'll just format weirdly.

feel free to open up a new PR if you want to change it again, or I can change it on the main branch whenever I get back to coding

**Pull request merged**